ophyd/ophydobj.py

```diff
@@ -10,6 +11,11 @@
 from .log import control_layer_logger
 
 
+OPHYD_DEBUG_WITH_INSPECT = os.getenv('OPHYD_DEBUG_WITH_INSPECT')
```

Comment:
Do you imagine this being the only thing we'll want to observe more closely in a "slow" debug build?
If not, we'll probably end up renaming this later- the naming is very specific to this one case.

Reply:
Alternatively, going the other direction and making it `OPHYD_DEBUG` could make it easier to switch on/off general debugging tools. I think @ZLLentz may have been pointing that way.